recipes/ollama/meta.yaml

```
@@ -0,0 +1,51 @@
+{% set name = "ollama" %}
+{% set goname = "github.com/jmorganca/ollama" %}
+{% set version = "0.1.11" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  #url: https://{{ goname }}/archive/v{{ version }}.tar.gz
+  #sha256: 881d81d6a113f5361fc1648ee3f07f596b96a25ff9c9e6b243c2386adb8a0b26
+  git_url: https://github.com/jmorganca/ollama.git
+  git_rev: v{{ version }}
```

This project has vendor-ed dependencies that need git to work.
```suggestion
  #url: https://{{ goname }}/archive/v{{ version }}.tar.gz
  #sha256: 881d81d6a113f5361fc1648ee3f07f596b96a25ff9c9e6b243c2386adb8a0b26
  git_url: https://github.com/jmorganca/ollama.git
  git_rev: v{{ version }}
  git_depth: 1
```